sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -36,6 +39,13 @@
  */
 public class ClientLogger {
     private final Logger logger;
+    private final String className;
+
+    private static final String LOG_MESSAGE_FORMAT = "%s: %s %s%n%s";
```

Comment:
How did we decide on this log message format? Additionally, should we just have a format for logging stack traces as well instead of manually handling adding a newline and stack trace?

Reply:
Working on stack trace format.